src/main/java/ShopOrders.java

```diff
@@ -0,0 +1,34 @@
+import java.util.*;
+
+public class ShopOrders {
+    public static void main(String[] args) {
+
+        ArrayList<String> orders = new ArrayList<>();
+        orders.add("order1");
+        orders.add("order1");
+        orders.add("order2");
+        orders.add("order2");
+        orders.add("order3");
+        orders.add("order4");
+        orders.add("order5");
+
+        Set treeSet = new TreeSet(orders);
```

Comment:
Please provide the parametrized type for all Sets (e.g. `Set<String> treeSet = new TreeSet<>(orders);`)